src/App.scss

```
@@ -1307,3 +1307,102 @@ body {
+// ========================================
+.did-tooltip-popup {
+  .maplibregl-popup-content {
+    background: rgba(20, 20, 30, 0.95);
+    backdrop-filter: blur(12px);
+    -webkit-backdrop-filter: blur(12px);
+    border: 1px solid rgba(255, 255, 255, 0.1);
+    border-radius: 8px;
+    padding: 0;
+    box-shadow: 0 8px 32px rgba(0, 0, 0, 0.4), 0 2px 8px rgba(0, 0, 0, 0.3);
+    color: #fff;
+    font-family: 'Inter', 'Noto Sans JP', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)